Gemfile.lock

```diff
@@ -210,60 +208,65 @@ GEM
       rb-inotify (~> 0.9, >= 0.9.7)
       ruby_dep (~> 1.2)
     mercenary (0.3.6)
-    mini_portile2 (2.3.0)
+    mini_portile2 (2.4.0)
     minima (2.4.0)
       jekyll (~> 3.5)
       jekyll-feed (~> 0.9)
       jekyll-seo-tag (~> 2.1)
-    minitest (5.11.3)
-    multipart-post (2.0.0)
-    net-dns (0.8.0)
-    nokogiri (1.8.5)
-      mini_portile2 (~> 2.3.0)
-    octokit (4.8.0)
+    minitest (5.14.0)
+    multipart-post (2.1.1)
+    net-dns (0.9.0)
+    nokogiri (1.10.8)
+      mini_portile2 (~> 2.4.0)
+    nokogumbo (2.0.2)
+      nokogiri (~> 1.8, >= 1.8.4)
+    octokit (4.16.0)
+      faraday (>= 0.9)
       sawyer (~> 0.8.0, >= 0.5.3)
-    parallel (1.12.1)
-    pathutil (0.16.1)
+    parallel (1.19.1)
+    pathutil (0.16.2)
       forwardable-extended (~> 2.6)
     public_suffix (2.0.5)
+    rainbow (3.0.0)
     rb-fsevent (0.10.3)
-    rb-inotify (0.9.10)
-      ffi (>= 0.5.0, < 2)
+    rb-inotify (0.10.1)
+      ffi (~> 1.0)
     rouge (2.2.1)
     ruby-enum (0.7.2)
       i18n
     ruby_dep (1.5.0)
-    rubyzip (2.0.0)
-    safe_yaml (1.0.4)
-    sass (3.5.5)
+    rubyzip (2.2.0)
+    safe_yaml (1.0.5)
+    sass (3.7.4)
       sass-listen (~> 4.0.0)
     sass-listen (4.0.0)
       rb-fsevent (~> 0.9, >= 0.9.4)
       rb-inotify (~> 0.9, >= 0.9.7)
-    sawyer (0.8.1)
-      addressable (>= 2.3.5, < 2.6)
-      faraday (~> 0.8, < 1.0)
+    sawyer (0.8.2)
+      addressable (>= 2.3.5)
+      faraday (> 0.8, < 2.0)
     terminal-table (1.8.0)
       unicode-display_width (~> 1.1, >= 1.1.1)
     thread_safe (0.3.6)
-    typhoeus (1.3.0)
+    typhoeus (1.3.1)
       ethon (>= 0.9.0)
-    tzinfo (1.2.5)
+    tzinfo (1.2.6)
       thread_safe (~> 0.1)
-    unicode-display_width (1.3.0)
-    yell (2.0.7)
+    unicode-display_width (1.6.1)
+    yell (2.2.2)
 
 PLATFORMS
   ruby
 
 DEPENDENCIES
-  eip_validator (>= 0.8.0)
+  ecip_validator (>= 0.9.0)
```

Comment:
Not the biggest fan of this being an external dep.

since a tool that validates ECIPs is kind of part of the ECIP spec itself. Who maintains this tool? Where is it located?

Reply:
> The new validator can be found here: https://github.com/meowsbits/ecip_validator and is published with rubygems here https://rubygems.org/gems/ecip_validator. As mentioned in #305, if there's interest in pushing forward a community-development version, I'd be happy to transfer it to github.com/ethereumclassic/.

Reply:
Yea, I can readily get on board with moving it closer to this repo... even into this repo. But I'm just thinking one step at a time, at least to use it as-is to get the ECIPs straightened out, since this is how EIPs do it, so it's an established pattern, and if there's energy to restructure/replace/relocate the tool then that's great but doesn't seem like a dependency for the spirit of this PR to me.

Reply:
And I do really think that would be something worth prioritizing here too, since that code will be just another representation of some of the parameters outlined in ECIP1000 or whatever, like requiring frontmatter fields, values, whatever. Actually even a better way of doing it than ECIP1000/whatever imo.